components/nexus-core/src/main/java/org/sonatype/nexus/configuration/application/DefaultNexusConfiguration.java

```diff
@@ -246,8 +292,7 @@ public synchronized void loadConfiguration(boolean force)
       eventBus.post(loadEvent);
 
       if (loadEvent.isVetoed()) {
-        log.info(
-            vetoFormatter.format(new VetoFormatterRequest(loadEvent, log.isDebugEnabled())));
+        log.info(new DefaultVetoFormatter().format(new VetoFormatterRequest(loadEvent, log.isDebugEnabled())));
```

Comment:
Since `DefaultVetoFormatter` has no state, you could create a static/member instance and use it for all events rather than create a new instance of the formatter over and over again.